src/styles/blocks/header.scss

```
@@ -15,6 +15,8 @@
 }
 
 .nav {
+  transition: transform 300ms;
```

```suggestion
  transition: color 300ms;
```

you changing color on hover so you need to specify this property